pkg/plugins/plugins.go

```diff
@@ -176,6 +176,10 @@ func ReadPluginJSON(reader io.Reader) (JSONData, error) {
 	for _, include := range plugin.Includes {
 		if include.Role == "" {
 			include.Role = org.RoleViewer
+			// Allow includes to users with access permission.
```

Comment:
This will allow access to routes with no role/action specification to users with `plugins.app:access`